notebooks/intro/project.ipynb

```diff
@@ -122,18 +122,18 @@
    "source": [
     "### Step 4: Have fun!\n",
     "\n",
-    "Start working towards your goal. If you find your project too difficult, get stuck, or simply get bored, ask for help in the [`#textbook-projects`](https://join.slack.com/share/enQtMjcxMjMyNzczMzg1Ni02ZDQ5NTAwZGIwMzJhOTQzMzUzZDNjOTg3ZTM3YzY2ZTRmZTRiMWMwZjZmMTczYTEyMTE0NThiMGNiOTFiOWQ0) channel. The mentors can either point you in the right direction, or help you adjust your project’s goals.\n",
+    "Start working towards your goal. If you find your project too difficult, get stuck, or simply get bored, ask for help in the [`#textbook-projects`](https://qiskit.slack.com/archives/C028AGU6KDX) channel. The mentors can either point you in the right direction, or help you adjust your project’s goals.\n",
     "\n",
     "### Step 5: Share your work\n",
     "\n",
-    "After you’ve put in the time and effort, share your work with others on the [`#textbook-projects`](https://join.slack.com/share/enQtMjcxMjMyNzczMzg1Ni02ZDQ5NTAwZGIwMzJhOTQzMzUzZDNjOTg3ZTM3YzY2ZTRmZTRiMWMwZjZmMTczYTEyMTE0NThiMGNiOTFiOWQ0) channel. If you maintain your GitHub profile and GitHub pages web page, you can use this to evidence your knowledge and experience to others (e.g. in applications or interviews)."
+    "After you’ve put in the time and effort, share your work with others on the [`#textbook-projects`](https://qiskit.slack.com/archives/C028AGU6KDX) channel. If you maintain your GitHub profile and GitHub pages web page, you can use this to evidence your knowledge and experience to others (e.g. in applications or interviews)."
    ]
   }
  ],
  "metadata": {
   "celltoolbar": "Edit Metadata",
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 3 (ipykernel)",
```

Comment:
Thanks for the contribution @divshacker ! Could you remove this change from the PR? We would like to maintain the changes only related with the new links 🙏